recipes/device-fingerprinting-pro/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "device-fingerprinting-pro" %}
+{% set version = "1.0.3" %}
+{% set python_min = "3.8" %}
```

Comment:
We typically don’t set this lower than the current one supported by conda forge. 3.8 is too low